mk/lib.mk

```diff
@@ -60,14 +60,14 @@ $(lib-libfile): $(objs)
 endif
 ifeq ($(CFG_ULIBS_SHARED),y)
 ifeq ($(sm)-$(CFG_TA_BTI),ta_arm64-y)
-lib-ldflags$(libuuid) += $$(call ld-option,-z force-bti) --fatal-warnings
+lib-ldflags$(lib-shlibfile) += $$(call ld-option,-z force-bti) --fatal-warnings
```

Comment:
` lib-shlibfile` is empty when `CFG_ULIBS_SHARED` isn't `y`

Reply:
Hmm, this comment would be more relevant below at line 70.

Reply:
Line 70 is stilll in the `ifeq ($(CFG_ULIBS_SHARED),y) .. endif` block

Reply:
My bad, you're right.